### PR TITLE
Several Enhancements

### LINKED ISSUE #76: Weird look on mobile
<details>
<summary>System Information</summary>

| Name            | Value                                                          |
| --------------- | -------------------------------------------------------------- |
| OS              | Fedora 30/Android                                              |
| Node Version    | 10.16.0                           |
| Project Version | 0.1.5 (latest commit as of this issue) |

</details>

<details>
<summary>What is the current behavior?</summary>

Screenshot_20190620-153624
Screenshot_20190620-153633

</details>

<details>
<summary>What is the expected behavior?</summary>

1) The image is still wrong, it should be whole size. Seems the CSS didn't fix it (#49)
2) The boxes are all different size and some have overflow and aren't centered
</details>

<details>
<summary>Logs</summary>

None relevant

```

```

</details>

### PR DESCRIPTION
**Pull request by Gelbpunkt**
1) Now using [prettier](https://prettier.io/) for code formatting
2) Added `npm run prettier`
3) Added #80 
4) Added #67 
5) Enhanced #63 and added a `social.json` file. It gets created when building and contains the social media data. `gitfolio update` will pull from it instead of `config.json`

I'm open for feedback :)

PS: this somehow also fixed #76 and of course #79

If the diff is unreadable, I suggest cloning and viewing it with your own eyes

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -1,85 +1,114 @@
...
 async function buildCommand(username, program) {
-    await populateCSS(program);
-    let sort = program.sort ? program.sort : 'created';
-    let order = program.order ? program.order : "asc";
-    let includeFork = program.fork ? true : false;
-    await populateConfig(sort, order, includeFork);
-    updateHTML(('%s', username), sort, order, includeFork);
+  await populateCSS(program);
+  await populateSocials();
+  const socials = await getSocials();
```
Comment (rohit-gohri) [inline build.js]: Could move the `getSocials` calls to updateHTML fn as both `build` and `update` are just passing the values to it.

Comment (Gelbpunkt) [reply build.js]: Might as well want to do that with the config, but I left that out for now

**Comment by Gelbpunkt** [comment]
Added a file called `ignore.json` to the dist. You can add repository names there and run `gitfolio update` to make them disappear from the list of your works. Closes #82

File: build.js (hunk 1 above)
Comment (Gelbpunkt) [reply build.js]: Done

**Comment by rohanmohapatra** [comment]
Much required Enhancement

**Pull request closed without merging**